includes/amp-helper-functions.php

```diff
@@ -603,6 +603,11 @@ function is_amp_endpoint() {
 		);
 	}
 
+	// note: is_embed() and is_feed() need $wp_query, so above checks must go first
```

Comment:
The Travis CI tests are currently on the following issue:
```
FILE: includes/amp-helper-functions.php
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 606 | ERROR | Inline comments must end in full-stops, exclamation marks, or
     |       | question marks
     |       | (Squiz.Commenting.InlineComment.InvalidEndChar)
--------------------------------------------------------------------------------
```
(see https://travis-ci.org/github/ampproject/amp-wp/jobs/674004902#L907-L914)

Until we fix this, we won't see whether the rest of the tests are passing.

Here's how we can fix this:
```suggestion
	// Note: is_embed() and is_feed() need $wp_query, so above checks must go first.
```

You can just choose to "Commit suggestion" to include this change, just make sure you give it an appropriate commit message instead of the GitHub default.

Reply:
I changed it and committed the change, is that enough?

Reply:
Yes. As you can see, the tests are passing now: https://travis-ci.org/github/ampproject/amp-wp/builds/674015488

Reply:
Great, thanks.